### ISSUE TITLE
Adjust wording on request publishing page

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#1519]
Users were unclear during testing why they couldn't publish yet; some thought it was because the product was still under development.

Be sure to indicate this only applied to the first time.

**Comment by connorscarolyns** [comment]
I assume it's still a problem; we didn't test the sign-up process in the last round. But the problem occurred on the onboarding screen and in the app. Users assumed they would have to unlock publishing every time they built a form. A small line about only having to do this once will be enough. 

Users were very concerned about doing this every time and indicated they would find another way to publish forms if this was the case. Clarifying from the start will go a long way to improve user confidence.

**Comment by anikbrazeau** [comment]
Thanks for clarifying! :D Turns out that is a bit more confusing than I initially thought because there are basically two ways to be blocked from publishing and they're separate but maybe a bit muddled: 

1. The ability of the account to publish (support unlocks -- which disappears after onboarding)
2. The checklist of requirements for each form (user has control -- which is more about the state of the form)

**Comment by connorscarolyns** [comment]
Correct, and to muddy it more... if you skipped the unlock step during onboarding, you'll be presented with the unlock publishing CTA when you land on the publishing checklist page.

**Comment by anikbrazeau** [comment]
<img width="1201" alt="Screenshot 2023-05-24 at 3 13 17 PM" src="https://github.com/cds-snc/platform-forms-client/assets/38330843/a1cbfef1-87b9-41d4-ae38-de1961f2661b">

**Comment by connorscarolyns** [comment]
Yes, that's the one. This is the screen where users who skipped the unlock step during onboarding thought they would have to request an unlock every time.

**Comment by srtalbot** [comment]
@anikbrazeau - did we ever end up adding any clarifying language to these screens?

(No problem if not, just doing some card clean-up)

**Comment by anikbrazeau** [comment]
Yes, we specified that you unlock `before publishing your first form`.
![Screenshot 2024-05-09 at 8 20 49 AM](https://github.com/cds-snc/platform-forms-client/assets/38330843/f894578b-8b02-4944-92c1-6de484388ce9)

We also applied the different visual styling to separate the `Unlock publishing` in the publish checklist if that step was skipped in onboarding and changed the CTA to `Request publishing ability`

![Screenshot 2024-05-09 at 8 22 27 AM](https://github.com/cds-snc/platform-forms-client/assets/38330843/008ae40a-fc37-45c9-b1a1-fb6478c74101)

We can do more if it's still confusing? But I haven't seen this issue recently?
